src/components/Login.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+import Button from "./Button/Button";
+
+const LoginButton = styled(Button)`
+  width: 100px;
+  height: 40px;
+  border: none;
+  border-radius: 10px;
+  background: #AEA5DF;
+  color: white;
+`;
+
+export default function Login() {
+  return (
+    <>
+      <LoginButton type="button" content="로그인" />
+    </>
+  );
```

Comment:
반환하는 컴포넌트가 하나이기 때문에 다음과 같이 작성하시면 됩니다!

```suggestion
  return (
      <LoginButton type="button" content="로그인" />
  );
```

Reply:
해당 사항만 수정하시면 될 것 같습니다! 고생하셨습니다!